onnx_tf/backend_tf_module.py

```diff
@@ -126,22 +99,25 @@ def __init__(self, node, backend):
     self.handlers = backend._get_handlers(opset=None)
     self.handler_variables = self._create_handlers_variables(dict())
 
-  def _create_handlers_variables(self, vars_dict):
+  def _create_handlers_variables(self, var_dict):
     if self.handlers:
       handler = self.handlers[self.node.domain].get(
           self.node.op_type,
           None) if self.node.domain in self.handlers else None
-      if handler and bool(
-          handler.get_req_vars_template(self.node, self.node.attrs)):
-        for v_name, v_template in handler.get_req_vars_template(
-            self.node, self.node.attrs).items():
-          v_init, v_shape = v_template
-          v_name = get_variable_name(self.node, v_name)
-          vars_dict[v_name] = tf.Variable(v_init,
-                                          dtype=v_init.dtype,
-                                          shape=v_shape,
-                                          name=v_name)
-    return vars_dict
+      var_dict = handler.create_variables(
+          self.node, dict(), var_dict,
+          self._callback_for_subgraph) if handler else var_dict
+    return var_dict
+
+  def _callback_for_subgraph(self, graph, init_dict, var_dict):
+    if self.handlers:
+      for node in graph.node:
+        handler = self.handlers[node.domain].get(
+            node.op_type, None) if node.domain in self.handlers else None
+        var_dict = handler.create_variables(
+            node, init_dict, var_dict,
+            self._callback_for_subgraph) if handler else var_dict
+    return var_dict
```

Comment:
Is it possible to have a generic function for both BackendTfModule and TFModule?